openapi/models/Tag.v1.yaml

```diff
@@ -0,0 +1,19 @@
+title: Tag
+type: object
+properties:
+  id:
+    type: integer
+  name:
+    type: string
+  color:
+    type: string
+  project:
+    type: string
+    example: 'http://localhost:3000/projects/1'
+    format: uri
+x-examples:
+  Example:
+    id: 1
+    name: Rust
+    type: language
```

Comment:
This does not really match with the definition above, Apart from this te rest looks ok!